### ISSUE TITLE
WCAG 2.2 Violation - There are improperly implemented custom combobox/listbox elements present within the interface.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#31]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- 

### Instances

- #32 

#### Further information:

1.
